hack/make/test-integration

```diff
@@ -9,7 +9,7 @@ fi
 
 source hack/make/.integration-test-helpers
 
-if [ ! -z "${TEST_SKIP_INTEGRATION}" ] && [ ! -z "${TEST_SKIP_INTEGRATION_CLI}" ]; then
+if [[ "${TEST_SKIP_INTEGRATION}" == "1" ]] && [[ "${TEST_SKIP_INTEGRATION_CLI}" == "1" ]]; then
```

Comment:
Was there a reason for changing these to `[[` / `]]` ?

Note that (on request); we try to avoid the bash-specific `[[ ]]` where possibly; https://github.com/moby/moby/pull/38424#issuecomment-452882054

This could be the;

```suggestion
if [ -n "${TEST_SKIP_INTEGRATION}" ] && [ -n "${TEST_SKIP_INTEGRATION_CLI}" ]; then
```